### ISSUE TITLE
top-down proteomics

### ISSUE THREAD

**Issue by mwalzer** [HUPO-PSI/psi-ms-CV#144]
### Describe the new term or terms you would like to add.

I noticed that we do not have a CV term for TD-MS, here is what I suggest for that
```
id: MS:100????
name: top-down proteomics 
def: "A technique by which intact proteins are subjected to LC-MS/MS rather than enzymatic digests of proteins." [PSI:MS]
is_a: MS:1000857 ! run attribute
is_a: MS:1002489 ! special processing
synonym: "TD-MS" EXACT []
```

**Comment by chambm** [comment]
Where would this be used?

**Comment by mobiusklein** [comment]
This was w.r.t. a suggestion on the IMS mzML revision email thread. The main thesis there is that the "file contents" terms which describe the "type" of experiment could also be used to signal that an mzML file contains top-down spectra.

Excerpting:
> For mzML to accommodate deconvoluted top-down spectra, most of 3.4 of
the DIA/IMS recommendation applies. File content header,
processingMethod must of course be specified accordingly*, however I am
not sure how much detail can be expected for selectedIonList. @Kyowon:
Do you think it would be feasible to expect all ions involved by every
software to be reported?

>The main point of course are the deconvoluted spectra. I think one can
approach the accommodation of these spectra in two ways:
>1. View the charge deconvolution as a synthetic ion selection and
detection, hence add the new spectra as MS-level +1 (i.e. products) or
>2. Let the charge deconvolution produce a new spectrumList (because
afterall dataprocessing and sourceFileRef must be different)

My 2 bits:
Deconvolved spectra declare they are deconvolved by having a `charge` array or a `mass` array, and should have the relevant data processing actions in the header. If your goal is to store multiple processing states of the same spectrum inline, you would probably want to store the "main" representation in the standard arrays (m/z array, intensity array, etc.) and we could mint new array type terms to refer to the secondary array sets without losing the "meaning" of the spectrum for older mzML readers.

That said, trying to store multiple processing states of a spectrum together gets expensive, space-wise, especially if you try to preserve the profile spectrum. Additionally, if you're trying to maintain a high fidelity mapping, your deconvoluted spectra may not share a 1:1 representation with raw spectra.

Finally, deconvoluted mass spectra "mean" the same thing consistently, while with IMS or DIA, different vendors offer different paradigms that make the same set of data arrays mean different things, e.g. DIA-PASEF and Cyclic MSE are both DIA and include ion mobility, but how you interpret them could be different, and some strategies reflect vendor software specific tasks.


We do have a [working document ](https://docs.google.com/document/d/1T5UntSIs2HBkTJqEfSoORzkrequ2dTbjIwrXoW3HlYw/edit)for deconvoluted spectra in mzML which does add a few points that could be improved, but I haven't had that much time to work on this recently. There was also discussion around not just storing deconvoluted spectra for top-down, but also LC-MS features, which is a bigger problem that mzML isn't well suited to handle.

**Comment by mwalzer** [comment]
@chambm 
I think one good location for this would be in mzML's fileDescription/fileContent fields to mark them as TopDown data.

@mobiusklein 
Thanks for the working document link, can I add there? I don't want to scatter the discussion. Is there a dedicated group 'caring' for topdown data in mzML? Maybe we could use the discussion function of a new issue on topdown @ https://github.com/HUPO-PSI/mzML ?

**Comment by edeutsch** [comment]
Addressed in PR #183. Provide comments there @mwalzer 
closing.
